FFM-7811: adds ff server relnotes 1.1054.0 (#1700)
harness/developer-hub: release-notes/feature-flags.md

  <TabItem value="Fixed issues">
```

#### Feature Flags server

* Previously, re-enabling Git Experience did not trigger an immediate Git sync. With this change, flags are synchronized as soon as Git Experience is re-enabled. (FFM-7670)

* Some Git sync operations were failing if there was a large volume of flags and environments. This fix increased the transaction time-out for Git sync calls to allow processing of a large number of the flags. (FFM-7638)

* Policy checks weren't being carried out on flag rules added from the Target Group details page. This issue has been fixed. (FFM-7607)(FFM-7606)

* Previously, Feature Flag permissions and roles assigned to users or user groups in Access Control were applied at the account and project levels, but not at the organization level. With this change, roles and permissions assigned at the organization level are now honored. (FFM-7376)

> * When trying to delete a flag that is a prerequisite to another flag, a new explanatory message is now displayed for why this cannot be done: *Cannot delete flag which is a prerequisite for other flags*. (FFM-5105)

Reviewer: ```suggestion
* When you try to delete a flag that is a prerequisite to another flag, the UI now shows a message that explains why this cannot be done: *Cannot delete flag which is a prerequisite for other flags*. (FFM-5105)
```
